Tests/Marketplace/upload_packs.py

```diff
@@ -1204,16 +1177,14 @@ def main():
         task_status, is_missing_dependencies = pack.format_metadata(index_folder_path,
                                                                     packs_dependencies_mapping, build_number,
                                                                     current_commit_hash, pack_was_modified,
-                                                                    statistics_handler, pack_names)
+                                                                    statistics_handler, pack_names, id_set, marketplace)
 
         if is_missing_dependencies:
-            # If the pack is dependent on a new pack
-            # (which is not yet in the index.zip as it might not have been iterated yet)
-            # we will note that it is missing dependencies.
-            # And finally after updating all the packages in index.zip - i.e. the new pack exists now.
-            # We will go over the pack again to add what was missing.
+            # If the pack is dependent on a new pack, therefore it is not yet in the index.zip as it might not have
+            # been iterated yet, we will note that it is missing dependencies, and after updating the index.zip with
+            # all new packs - we will go over the pack again to add what was missing.
```

Comment:
```suggestion
            # all new packs - we will go over the packs again to add what was missing.
```

Reply:
This is actually pack - we go over this specific pack again (the pack of this current iteration)